README.md

```diff
@@ -3,11 +3,12 @@
 Replace this readme with your own information about your project.
 
 Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+Create a survey containing minumum 3 questions, radio buttons, select dropdown and a submit button that displays the summary of the users input.
 
 ## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+UseState and how to use useState in the different components, Took some time connecting summary to submit f.ex. The required input stopped working.
 
```

Comment:
Maybe add som info how you have tested web accessibility as that was a requirement for this project e.g I used ChromeVox as a screen reader.